lib/code_corps/emails/project_user_acceptance_email.ex

```diff
@@ -0,0 +1,35 @@
+defmodule CodeCorps.Emails.ProjectUserAcceptanceEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.{Project, ProjectUser, Repo, User}
+  alias CodeCorps.Emails.BaseEmail
+  alias CodeCorps.Presenters.ImagePresenter
+
+  def create(%ProjectUser{project: project, user: user}) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), build_model(project, user))
+  end
+
+  defp build_model(%Project{} = project, %User{} = user) do
+    %{
+      project_logo: ImagePresenter.large(project),
+      project_title: project.title,
+      project_url: project |> preload() |> url(),
+      subject: "TODO: Put subject here",
```

Comment:
You still need a subject. I've added this field because I believe Postmark is expecting it. If that's wrong, feel free to remove it. In hindsight, I should've not allowed the test to pass here.